src/data/teams.ts

```diff
@@ -0,0 +1,64 @@
+import { Team } from "../types";
+
+export const Teams: Team[] = [
+  {
+    description:
+      "This team supports and enables everyone else to be able to do what needs to be done. They are also responsible for the high level decisions that chart the overall course of the project.",
+    members: [
+      ["ReillyBrogan", null],
+      ["JoeyRiches", null],
+      ["TraceyClark", null],
+      ["TroyHarvey", null],
+      ["DavidHarder", null],
+    ],
+    name: "Admin Team",
+  },
+  {
+    description:
+      "This team handles the building of packages in the Solus repository. Some team members help maintain one or more of our included desktop environments.",
+    members: [
+      ["ReillyBrogan", "Plasma"],
+      ["EvanMaddock", "Budgie/Xfce"],
+      ["JoeyRiches", "GNOME"],
+      ["SilkeHofstra", null],
+      ["TraceyClark", null],
+      ["DavidHarder", null],
+      ["TroyHarvey", null],
+      ["AlgentAlbrahimi", null],
+      ["GavinZhao", null],
+      ["HansKelson", null],
+      ["Malfisya", "GNOME"],
+    ],
+    name: "Packaging Team",
+  },
+  {
+    description: "This team writes and maintains software for Solus, such as eopkg, ypkg, and solbuild.",
+    members: [
+      ["ReillyBrogan", null],
+      ["SilkeHofstra", null],
+      ["JoeyRiches", null],
+      ["HansKelson", null],
+    ],
+    name: "Development Team",
+  },
+  {
+    description:
+      "This team is responsible for ensuring that the Solus infrastructure and services are kept up to date and running smoothly.",
+    members: [
+      ["ReillyBrogan", null],
+      ["SilkeHofstra", null],
```

Comment:
Harvey should be added to this list
https://github.com/orgs/getsolus/teams/infrastructure-team